### PR TITLE
Create new graph fragment

### LINKED ISSUE #139: Add graph fragment
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What you expected to happen**:
Really rough idea. Provide a stats fragment for open startup stats and or similar

**Anything else we need to know?**:
https://www.openstartups.net/

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Add new graph fragment

**Which issue this PR fixes**:
fixes #139 

**Special notes for your reviewer**:
No idea why build files are changing. I hate having them committed.

**Release note**:
```release-note
- graph: New graph fragment available. 
```

### CODE REVIEW

File: exampleSite/content/fragments/stats/stats/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++
+fragment = "graph"
+weight = 110
+background = "secondary"
+
+title = "Graph Fragment"
```
Comment (stp-ip) [inline exampleSite/content/fragments/stats/stats/index.md]: `subtitle` is missing

File: exampleSite/content/fragments/stats/code-stats.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
++++
+
+<details><summary>Code (index)</summary>
+```
++++
+fragment = "graph"
+weight = 110
+background = "secondary"
+
+title = "Graph Fragment"
```
[the remark this answers is not in the document]
Comment (stp-ip) [reply exampleSite/content/fragments/stats/code-stats.md]: `Beautifully showcase your data with graphs based on charts.js`

**Comment by mpourismaiel** [comment]
I think a more descriptive content would be good.

**Comment by stp-ip** [comment]
Let's merge the implementation and maybe add production users of Syna in a new PR as data. That would be interesting.

**Pull request merged**